### PR TITLE
Fix two bugs for template type-checking

### PR DESCRIPTION
**Pull request by alxhub**

### CODE REVIEW

**Comment by alxhub** [comment]
[Presubmit](http://test/OCL:245093156:BASE:245093184:1556133717242:d89718e9)

File: packages/compiler-cli/src/ngtsc/typecheck/src/type_check_file.ts [hunk 1]
```diff
@@ -61,6 +61,7 @@ export class TypeCheckFile extends Environment {
     for (const stmt of this.tcbStatements) {
       source += printer.printNode(ts.EmitHint.Unspecified, stmt, this.contextFile) + '\n';
     }
+    console.error(source);
```
Comment (AndrewKushnir) [inline packages/compiler-cli/src/ngtsc/typecheck/src/type_check_file.ts]: Is this `console.error` added intentional? If so, may be add a comment?

File: packages/compiler-cli/test/ngtsc/template_typecheck_spec.ts [hunk 2]
```diff
@@ -148,6 +148,29 @@ describe('ngtsc type checking', () => {
...
+    }
+
+    @NgModule({
+      declarations: [TestCmp],
+      imports: [CommonModule],
+    })
+    export class Module {}
+    `);
+
+    env.driveMain();
```
Comment (AndrewKushnir) [inline packages/compiler-cli/test/ngtsc/template_typecheck_spec.ts]: May be add a comment that we don't need any additional assertions here (i.e. we should not throw)?

**Review by AndrewKushnir — APPROVED** [review]
LGTM 👍 Just left a couple nits. Thank you.

File: packages/compiler-cli/src/ngtsc/typecheck/src/type_check_file.ts (hunk 1 above)
Comment (alxhub) [reply packages/compiler-cli/src/ngtsc/typecheck/src/type_check_file.ts]: Deleted.

File: packages/compiler-cli/test/ngtsc/template_typecheck_spec.ts (hunk 2 above)
Comment (alxhub) [reply packages/compiler-cli/test/ngtsc/template_typecheck_spec.ts]: This is a common pattern in the template type-checking tests - I think it's okay.

**Comment by alxhub** [comment]
Caretaker: I am the approver :)

**Pull request closed without merging**